src/ts/legendControl.ts

```diff
@@ -85,9 +85,9 @@ module powerbi.extensibility.visual {
                 const valueElement = document.createElement('span');
                 valueElement.className = 'mapbox-legend-value middle'
                 if (typeof colorStop === "number") {
-                    valueElement.innerHTML = valueFormatter.format(colorStop, format || LegendControl.DEFAULT_NUMBER_FORMAT);
+                    valueElement.innerHTML = mapboxUtils.sanitizeHTML(valueFormatter.format(colorStop, format || LegendControl.DEFAULT_NUMBER_FORMAT));
```

Comment:
you should avoid innerHTML use, or you risk additional attacks. Any reason why not to use standard apis?

Reply:
> Any reason why not to use standard apis?

Good feedback - yes it's easy enough to avoid innerHTML altogether.  Fixing in next commit.